systemd/ua-status.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ubuntu Advantage status refresh
```

Comment:
We should add:

After=network-online.target
Want=network-online.target

as ua status is a network hit.

Reply:
thanks done.